crates/rnote-ui/src/canvas/input.rs

```diff
@@ -168,6 +172,13 @@ pub(crate) fn handle_pointer_controller_event(
         for (element, event_time) in elements {
             tracing::trace!("handle pen event element - element: {element:?}, pen_state: {pen_state:?}, event_time_delta: {:?}, modifier_keys: {modifier_keys:?}, pen_mode: {pen_mode:?}", now.duration_since(event_time));
 
+            #[cfg(target_os = "windows")]
+            {
+                if element.pressure > 0.0 && is_stylus {
+                    pen_state = PenState::Down;
+                }
+            }
+
```

Comment:
Here too - let's document it also in code

Reply:
fixed